Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: retool
 description: A Helm chart for Kubernetes
 type: application
-version: 4.12.1
+version: 5.0.0
```

Comment:
did we figure out why us pinning the dependent chart version didn't work? i didn't see that they got rid of the version we had pinned last i checked

Reply:
do you mean why 4.11.9 is associated with postgresql version 10.4.3 even though we pinned it to 12.1.5 in 4.11.8 release of chart? not yet, i'm working on fixing this issue so customers can work with the latest retool-helm version first. will probably download the earlier version tgz files and dissect later.

Reply:
oh my bad i missed the part where we pinned it to 12.1.5, makes sense